e2etests/testdata/cel-config.yaml

```diff
@@ -0,0 +1,23 @@
+checks:
+  addAllBuiltIn: false
+customChecks:
+  - name: "cel-forbidden-annotation"
+    description: "cel espression cehck"
+    remediation: "Remove reloader.stakater.com/auto annotation"
```

Comment:
I don't know what stakater reloader is, but it feels a bit weird to use it as an example here. Let's replace it with a more standard `foo.bar/baz` annotation